ntbs-integration-tests/NotificationPages/TreatmentEventsEditPageTests.cs

```diff
@@ -331,25 +331,28 @@ public async Task PostNewTreatmentOutcomeEventNoOutcome_ReturnsValidationError()
         }
 
 
-        [Theory]
-        [InlineData("hello")]
-        [InlineData("27")]
-        [InlineData("")]
-        public async Task WhenEmptyOrInvalid_ValidateSelectedTreatmentOutcomeTypeProperty_ReturnsExpectedInvalidResponse(string value)
+        [Fact]
+        public async Task WhenEmptyOrInvalid_ValidateSelectedTreatmentOutcomeTypeProperty_ReturnsExpectedInvalidResponse()
```

Comment:
Have we reduced test coverage here? Doesn't it still make sense to test the invalid integer?

Reply:
I have reverted these changes, with the addition of null case as this is the behaviour we are actually expected on an empty select.